packages/block-editor/src/components/inserter/block-patterns-tab.js

```diff
@@ -93,9 +102,20 @@ function usePatternsCategories( rootClientId ) {
 				label: _x( 'Uncategorized' ),
 			} );
 		}
+		if ( filteredUnsyncedPatterns.length > 0 ) {
+			categories.push( {
+				name: 'reusable',
+				label: _x( 'Your patterns' ),
```

Comment:
I think it may be worth reconsidering this label, as it won't make much sense for sites with multiple users.

Maybe 'User patterns'? cc @SaxonF 

The category name could also be changed to match, though it's a very minor implementation detail so not absolutely necessary.

Reply:
I like @jameskoster suggestion of "Custom patterns" which matches templates

Reply:
Should we also make the same change of wording in the Library's navigation screen then @SaxonF?

Reply:
@aaronrobertshaw yep !